src/components/Login/index.js

```diff
@@ -5,36 +5,65 @@ import {connect} from 'react-redux'
 import {firebaseConnect, isLoaded, isEmpty} from 'react-redux-firebase'
 import Lottie from 'react-lottie';
 import * as animationData from "../../pool";
-import {get} from 'lodash'
+import styled from 'styled-components'
+import {FadeIn} from 'animate-css-styled-components';
 
 const defaultOptions = {
   loop: false,
   autoplay: true,
   animationData: animationData
 };
 
+const Login = styled.div`
+  display:flex;
+  flex-direction: column;
+  justify-items: center;
+  align-items:center;
+`;
+
+export const Button = styled.button`
+  display: flex;
+  justify-content: center;
+  padding: 12px 12px;
+  margin:10px;
+  font-size: 120%;
+  cursor: pointer;
+  border: 0 none;
+  border-radius: 4px;
+  color: black;
+  background: rgba(255,255,255,0.5);
+
+  :hover {
+    background-color: #517fa4;
+    transition: all 150ms linear;
+  }
+`;
+
 export const LoginPage = ({firebase, auth}) => {
   return (
     <div>
-      {isLoaded(auth) && isEmpty(auth)
-        ? <button
-          onClick={() => firebase.login({provider: 'google', type: 'popup'})}
-        >Login With Google
-        </button>
-        :
-        <button
-          onClick={() => firebase.logout()}
-        >Log out
-        </button>
-      }
-      <div>
+      <Login>
```

Comment:
It’s nice bit all the logic inside the html makes it hard to read.
IMHO it’s better to do it before the return. Like:
Var logic = test ? Value1 : value2;

Return ( <htlm> {logic} </html> )

Sorry for the bad format I’m typing on my phone 😂

Reply:
Jep, it was a bit messy! Moved it up.